event_multi_image/models/event.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models
+
+
+class EventEvent(models.Model):
+    _name = "event.event"
+    _inherit = [_name, "multi_image_base.owner"]
```

Comment:
Oh sorry I misunderstood your question :sweat_smile:. The point is that AFAIK to extend an already-existing module with some another you need to declare it both in `_name` and in `_inherit`. You can see the example in [crm_mass_mailing](https://github.com/odoo/odoo/blob/8.0/addons/crm_mass_mailing/mass_mailing.py#L6).